arduino/smartcar/vehicle_config.json

```diff
@@ -1,83 +1,83 @@
-{
-   "vehicle": "RayTank",
-   "slots": {
-      "Front": {
-         "class": "AnalogRaycast",
-         "name": "Front Infrared",
-         "props": {
-            "pin": 0
-         }
-      },
-      "Front2": {
-         "class": "Camera",
-         "name": "Front Camera",
-         "props": {
-            "pins": [
-               10,
-               1,
-               0,
-               2,
-               3,
-               5,
-               6,
-               4
-            ]
-         }
-      },
-      "FrontTop": {
-         "class": "UltraSonic",
-         "name": "Front UltraSonic",
-         "props": {
-            "trigger_pin": 6,
-            "echo_pin": 7
-         }
-      },
-      "Left": {
-         "class": "UltraSonic",
-         "name": "Left UltraSonic",
-         "props": {
-           "trigger_pin": 10,
-           "echo_pin": 1
-         }
-      },
-      "Right": {
-         "class": "UltraSonic",
-         "name": "Right UltraSonic",
-         "props": {
-           "trigger_pin": 11,
-           "echo_pin": 2
-         }
-      },
-      "Back": {
-         "class": "UltraSonic",
-         "name": "Back UltraSonic",
-         "props": {
-           "trigger_pin": 15,
-           "echo_pin": 3
-         }
-      }
-   },
-   "builtin": {
-      "Left BrushedMotor": {
-         "forward_pin": 12,
-         "backward_pin": 14,
-         "enable_pin": 13
-      },
-      "Right BrushedMotor": {
-         "forward_pin": 25,
-         "backward_pin": 26,
-         "enable_pin": 27
-      },
-      "Gyroscope": {
-         "pin": 205
-      },
-      "Left Odometer": {
-         "distance_pin": 35,
-         "direction_pin": 34
-      },
-      "Right Odometer": {
-         "distance_pin": 36,
-         "direction_pin": 39
-      }
-   }
-}
\ No newline at end of file
+// {
```

Comment:
Some commented out code here. Do we keep it?

Reply:
Oh I forgot about this. I think we can remove the file since it was only used to make the emulator car into a tank. I'll remove it

Reply:
Deleted.